packages/e2e-tests/specs/editor/various/allowed-patterns.test.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	activatePlugin,
+	createNewPost,
+	deactivatePlugin,
+	searchForPattern,
+	toggleGlobalBlockInserter,
+} from '@wordpress/e2e-test-utils';
+
+const checkPatternExistance = async ( name, available = true ) => {
```

Comment:
Misspelling at _existence_:

```suggestion
const checkPatternExistence = async ( name, available = true ) => {
```

Reply:
Thank you!